core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -20,6 +20,7 @@
 from autogluon.common.utils.lite import disable_if_lite_mode
 from autogluon.common.utils.log_utils import DuplicateFilter
 from autogluon.common.utils.resource_utils import ResourceManager, RayResourceManager
+from autogluon.common.utils.resource_utils import get_resource_manager
```

Comment:
nit: can be on same line as L22

Reply:
I remember when using `black` it will reformat the import to saperate Class imported and function imported. That's why I did it here. I tried to search for the PEP rule but not sure what's the correct way to find it

Reply:
ok, not a big deal, I'm not familiar with the official guidelines here